### PR TITLE
feat(pattern): simple-overview alpha pattern creation

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Related Ticket(s)

Relative to ticket #727 

### Description

Creation of the alpha version of the Simple Overview pattern.

Screen Shot 2019-12-10 at 14 56 43

### Changelog

**New**

- alpha Simple Overview pattern

### CODE REVIEW

File: packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+          </div>
+          <div
+            className={classNames(
+              `${prefix}--simpleoverview__col`,
+              `${prefix}--simpleoverview__content-container`
+            )}>
+            <h5 className={`${prefix}--simpleoverview__heading`}>{heading}</h5>
+            <p className={`${prefix}--simpleoverview__content`}>{copy}</p>
+            {_renderLink(link)}
+          </div>
```
Comment (jeffchew) [inline packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js]: In the page hierarchy I'm wondering if this is an h2/h3? Any thoughts on this @kennylam @annawen1 @photodow @emerson-pereira @andysherman2121 @mkothur @carthick @sannminwin

File: packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        </div>
+      </div>
+    </section>
+  );
+};
+
+SimpleOverview.propTypes = {
+  label: PropTypes.string,
+  heading: PropTypes.string,
+  copy: PropTypes.string,
```
Comment (jeffchew) [inline packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js]: @RaphaelAmadeu any required props should be represented as such here

File: packages/styles/scss/ibm-dotcom-styles.scss [hunk 3]
```diff
@@ -28,6 +28,7 @@
 @import 'patterns/listsection/listsection';
 @import 'patterns/pictogramarray/pictogramarray';
 @import 'patterns/simplelongform/simplelongform';
+@import 'patterns/simplelongform/simpleoverview';
```
Comment (jeffchew) [inline packages/styles/scss/ibm-dotcom-styles.scss]: @RaphaelAmadeu is this the correct path?

File: packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js (hunk 1 above)
Comment (RaphaelAmadeu) [reply packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js]: This is a good topic, I think that as the H1 and H2  are commonly used in the Leadspace. Maybe a H3 and H4 would be more appropriate, as a second thought

File: packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js (hunk 2 above)
Comment (RaphaelAmadeu) [reply packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js]: I've fixed this is my latest commit, thanks!

File: packages/styles/scss/ibm-dotcom-styles.scss (hunk 3 above)
Comment (RaphaelAmadeu) [reply packages/styles/scss/ibm-dotcom-styles.scss]: I've fixed the path on my latest commit, thanks!

File: packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js (hunk 1 above)
Comment (jeffchew) [reply packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js]: @wonilsuhibm @larahanlon2 @oliviaflory where do you envision this pattern to typically be on a page?

Comment (wonilsuhibm) [reply packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js]: @jeffchew @sannminwin yes. This pattern will typically be right below the lead space pattern. Lead space presents the super high level overview and entices the user, then this pattern give a slightly deeper and better over view of the content of the page.

Comment (emerson-pereira) [reply packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js]: based on what wonil pointed out i'll agree with rapha, i think h3/h4 would make more sense

File: packages/patterns-react/src/patterns/SimpleOverview/__stories__/SimpleOverview.stories.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      const link = () => {
+        if (linkActive) {
+          return {
+            copy: text('Link copy:', 'Lorem Ipsum'),
+            href: text('Link href:', 'https://www.example.com'),
+            target: select('Link target:', targets, targets.blank),
+          };
+        } else {
+          false;
+        }
```
Comment (emerson-pereira) [inline packages/patterns-react/src/patterns/SimpleOverview/__stories__/SimpleOverview.stories.js]: @RaphaelAmadeu is this meant to return the false value? if so missing the actual `return` statement

Comment (RaphaelAmadeu) [reply packages/patterns-react/src/patterns/SimpleOverview/__stories__/SimpleOverview.stories.js]: This was fixed on my last commit, thanks!

File: packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js (hunk 1 above)
Comment (RaphaelAmadeu) [reply packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js]: I've changed it to H3 and H4 in my last commit.

**Pull request merged**